### PR TITLE
Data release

### PR DESCRIPTION
**Pull request by hareesh-ms**

### CODE REVIEW

File: internal/server/v2/facet/golden/contained_in_facet/country.json [hunk 1]
```diff
@@ -219,11 +162,6 @@
     }
   },
   "facets": {
-    "1042817753": {
```
Comment (hareesh-ms) [inline internal/server/v2/facet/golden/contained_in_facet/country.json:222]: deletion for deprecated imports.

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json [hunk 2]
```diff
@@ -3049,6 +3049,9 @@
         "Count_HousingUnit_2010To2014Date_OccupiedHousingUnit",
         "Count_HousingUnit_2010To2014Date_OwnerOccupied",
         "Count_HousingUnit_2010To2014Date_RenterOccupied",
+        "Count_HousingUnit_2010To2017Date_OccupiedHousingUnit",
```
Comment (beets) [inline internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json:3052]: were there some census updates in this release too?

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json [hunk 3]
```diff
@@ -6022,17 +6037,24 @@
         "Count_Person_18OrMoreYears_Employed_InLaborForce_NoHealthInsurance",
         "Count_Person_18OrMoreYears_Employed_InLaborForce_WithHealthInsurance",
         "Count_Person_18OrMoreYears_Female",
+        "Count_Person_18OrMoreYears_FoodInsecurity_AsAFractionOf_Count_Person_18OrMoreYears",
```
Comment (beets) [inline internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json:6040]: is this addition expected?

File: internal/server/v1/page/golden/place_page/ca.Health.json [hunk 4]
```diff
@@ -1909,79 +1909,91 @@
...
+            "2021": 9.7,
+            "2022": 10.9
           },
           "metadata": {
             "importName": "CDC500",
             "provenanceUrl": "https://www.cdc.gov/places/index.html",
             "measurementMethod": "AgeAdjustedPrevalence",
-            "observationPeriod": "P1Y"
+            "observationPeriod": "P1Y",
+            "scalingFactor": "100",
```
Comment (beets) [inline internal/server/v1/page/golden/place_page/ca.Health.json:1980]: lets verify that this results in the right chart render in the release

**Review by beets — APPROVED** [review]
thanks for the update. please keep an eye out on some of these changes when we do the website release @chejennifer @hareesh-ms

**Comment by chejennifer** [comment]
> thanks for the update. please keep an eye out on some of these changes when we do the website release @chejennifer @hareesh-ms

taking a look at the [screenshot diffs](https://autopush.datacommons.org/screenshot/compare/2024_10_07_17_00_01...2024_10_17_01_00_01?domain=autopush.datacommons.org), it looks like we've lost data for
- Amount_Debt_Government ([autopush](https://autopush.datacommons.org/tools/statvar#sv=Amount_Debt_Government) vs [prod](https://datacommons.org/tools/statvar#sv=Amount_Debt_Government))
- UnemploymentRate_Person_Male for USA ([autopush](https://autopush.datacommons.org/tools/statvar#sv=UnemploymentRate_Person_Male) vs [prod](https://datacommons.org/tools/statvar#sv=UnemploymentRate_Person_Male))
- UnemploymentRate_Person_Female for USA ([autopush](https://autopush.datacommons.org/tools/statvar#sv=UnemploymentRate_Person_Female) vs [prod](https://datacommons.org/tools/statvar#sv=UnemploymentRate_Person_Female))

is this expected?

**Comment by beets** [comment]
Good catch Jennifer - Eurostat deprecation but AustraliaStatistics isn't..

**Comment by chejennifer** [comment]
> * prod

oh, looks like the unemployment rate ones are also from eurostat deprecation, so probably expected then?

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json (hunk 2 above)
Comment (hareesh-ms) [reply internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json:3052]: There was subject table(S2502) CL submission which got built in the last UScensus run.

File: internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json (hunk 3 above)
Comment (hareesh-ms) [reply internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json:6040]: New variable as part of CDC500 import.

**Pull request merged**